library/erlang

```diff
@@ -18,6 +18,11 @@ Architectures: amd64, arm64v8, i386, s390x, ppc64le
 GitCommit: f0fffe2ba666a5f31e1de81d80dacdbf405c8e32
 Directory: 21/alpine
 
+Tags: 21.0.3-jessie, 21.0-jessie, 21-jessie, jessie
+Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
```

Comment:
Debian Jessie is no longer supported on all these architectures since it has transitioned to the Debian LTS team: https://github.com/docker-library/official-images/blob/9647dd55a72b9cb65d1fe71cd301525745a8b40d/library/debian#L45-L46

See also https://wiki.debian.org/LTS